src/components/settings/OptionItem.tsx

```diff
@@ -7,27 +7,32 @@ import { InputNumberField } from "../form/InputNumberField";
 
 type OptionItemProps = {
   option: OptionsItem;
+  noOverlay?: boolean;
+  overlayId?: string;
 };
 
-export const OptionItem = ({ option }: OptionItemProps) => {
-  const { type, iD, description, label, shortname, value } = option;
+export const OptionItem = ({ option, noOverlay, overlayId }: OptionItemProps) => {
+  const { type, iD, description, label, shortname, value, overlayed } = option;
```

Comment:
shortname is unused

Reply:
same as before... this could be done in some refactoring by someone who knows what he is doing better than me